sensor_msgs/Image im_narrow
sensor_msgs/Image im_wide